distribution/kubernetes/metastore-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: quickwit
+      service: metastore
+  template:
+    metadata:
+      labels:
+        app: quickwit
+        service: metastore
+    spec:
+      containers:
+        - name: quickwit
+          image: quickwit/quickwit:edge
+          args: ["run", "--service", "metastore"]
+          ports:
+            - containerPort: 7280
+              name: rest
+            - containerPort: 7281
+              name: grpc
+            - containerPort: 7282
```

Comment:
why we not the default ports?

Reply:
Kube cannot do TCP and UDP on the same port, so I set the gossip port on `7282`. Because of that, I chose to set the other ports explicitly.